extra/release_tool.py

```diff
@@ -452,10 +452,6 @@ def filter_docker_compose_files_list(list, version):
     _DOCKER_ONLY_YML = [
         "docker-compose.yml",
         "docker-compose.enterprise.yml",
-        "docker-compose.auditlogs.yml",
-        "docker-compose.connect.yml",
-        "docker-compose.config.yml",
-        "docker-compose.monitor.yml",
```

Comment:
As pointed out privately in Slack, these lines cannot be removed. This is used to parse all compose files in all references, including old release branches and tags which will still have these files.

Reply:
got it, fixed